upload-async/Sources/App/Request+FileIO.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2021-2021 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HummingbirdFoundation
+
+extension HBRequest {
+    var fileIO: HBFileIO { HBFileIO(application: self.application) }
```

Comment:
Is this really necessary? I'm sure it would be more obvious what you are doing if you just create an `HBFileIO` in the route handler

Reply:
Removed

Reply:
Hey @mcritz I'm not seeing any of these changes